sonyc_kalman/train_pykalmen.py

```diff
@@ -0,0 +1,107 @@
+import argparse
+import sys
+import os
+import pickle
+
+import pykalman as pk
+from numpy import ma
+from data import load_openl3_time_series as load_data
+
+
+DEFAULT_SENSOR = '/beegfs/work/sonyc/features/openl3/2017/sonycnode-b827ebefb215.sonyc_features_openl3.h5'
+
+def process_arguments(args):
+    parser = argparse.ArgumentParser()
+
+    parser.add_argument('--sensor',
```

Comment:
We should be consistent in our naming convention for this argument. Though, since we will be changing the way we're loading the data to use a train/valid split provided by @tomxi, it doesn't really matter at the moment.